subprojects/platform-jvm/src/integTest/groovy/org/gradle/jvm/toolchain/JavaToolchainDownloadSpiIntegrationTest.groovy

```diff
@@ -369,6 +369,85 @@ class JavaToolchainDownloadSpiIntegrationTest extends AbstractJavaToolchainDownl
         failure.getOutput().contains("Explicitly requested toolchains: [useless1, useless3].")
     }
 
+    def "cannot mutate repository rules after settings have been evaluated"() {
```

Comment:
This seems to introduce duplicate tests, there should be no changes to this file